x-pack/plugins/enterprise_search/public/applications/shared/version.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SemVer } from 'semver';
+import pkg from '../../../../../../package.json';
+
+export const CURRENT_VERSION = new SemVer(pkg.version as string);
+export const CURRENT_MAJOR_VERSION = CURRENT_VERSION.major;
```

Comment:
Should we consider moving this to `common/version.ts` since it's sorta feels like belongs next to `constants.ts`? 🤔 (no strong preference either way, just thinking out loud!)

Reply:
That's absolutely the best place for this. I forgot we had that folder. Will move it now. Thanks!

Reply:
@constancecchen I can’t find a way in the code base where this solution is being used outside of a constructor:

> If you grep initializerContext.env.packageInfo.version you will see examples of usage in other plugins.

Even we were able to put it on the context the place it’s used is just a simple utility that houses route strings so I don’t want to create a hacky `.tsx` component just for this. 

Thoughts?